docs/src/ex/0_Introduction.md

```diff
@@ -0,0 +1,178 @@
+
+Familiarity with Julia is assumed through out this notebook. If you are new to Julia, [Learning Julia](https://julialang.org/learning/) is a good starting point.
+
+For users new to Bayesian machine learning, please consider more thorough introductions to the field, such as [Pattern Recognition and Machine Learning](https://www.springer.com/us/book/9780387310732). This notebook tries to provide an intuition for Bayesian inference and gives a simple example on how to use Turing. Note that this notebook is not a comprehensive introduction to Bayesian machine learning.
+
+## Coin Flipping without Turing
+The following example aims to illustrate the effect of updating our beliefs with every piece of new evidence we observe. In particular, we will assume that we are unsure about the probability of heads in a coin flip. To get an intuitive understanding of what "updating our beliefs" is, we will visualize the probability of heads in a coin flip after each observed evidence.
+
+First, let's load some of the packages we are going to need to flip a coin (`Random`, `Distributions`) and show our results (`Plots`). You will note that **Turing** is not an import here -- we are not going to need it for this example. If you are already familiar with posterior updates, you can proceed to the next step.
+
+```julia
+# using Base modules
+using Random
+
+# load a plotting library
+using Plots
+
+# load the distributions library
+using Distributions
+```
+
+Next, we configure our posterior update model. First, let's set the true probability that any coin flip will turn up heads and set the number of coin flips we will show our model:
+
+```julia
+# set the true probability of heads in a coin
+p_true = 0.5
+
+# iterate from having seen 0 observations to 100 observations
+Ns = 0:100;
+```
+
+We will use the Bernoulli distribution to flip 100 coins, and collect the results in a variable called `data`.
+
+```julia
+# draw data from a Bernoulli distribution, i.e. draw heads or tails
+Random.seed!(12)
+data = rand(Bernoulli(p_true), last(Ns))
+
+# here's what the first five coin flips look like:
+data[1:5]
+```
+
+After flipping all our coins, we want to set a prior belief about what we think the distribution of coinflips look like. In our case, we are going to choose a common prior distribution called the [Beta](https://en.wikipedia.org/wiki/Beta_distribution) distribution. We will allow this distribution to change as we let our model see more evidence of coin flips.
+
+```julia
+# our prior belief about the probability of heads in a coin toss
+prior_belief = Beta(1, 1);
+```
+
+With our priors set and our data at hand, we can finally run our simple posterior update model.
```

Comment:
Maybe something along the lines of "... we can perform Bayesian inference for our model."

Reply:
Looks good to me, updated.